### PR TITLE
Update media-sound/flac123 to 0.0.12 from 0.0.11

### PR DESCRIPTION
**Pull request by Archetypic**

### CODE REVIEW

**Comment by tgurr** [comment]
**404: Not Found.** It doesn't even fetch for me, the tarball for 0.0.12 carries an additional **-release** in its name so the exheres has to reflect that as well.

**Comment by Archetypic** [comment]
Which file will tell me the `supported_autoconf` as well as `supported_automake` versions?

**Comment by tgurr** [comment]
> Which file will tell me the `supported_autoconf` as well as `supported_automake` versions?

Most recent versions should go first `1.14 1.15 1.16` -> `1.16 1.15 1.14`.

Only the most recent version for each should be added if it works with that. Usually when you see more than one version the package may be older so it has been added when 1.14 was the most recent one and then 1.15 and 1.16 got added after things were confirmed working with these newer versions.

**Comment by Archetypic** [comment]
Thank you for your assistance and patience.

File: packages/media-sound/flac123/flac123-0.0.12.exheres-0 [hunk 1]
```diff
@@ -1,21 +1,16 @@
 # Copyright 2009 Ali Polatel <alip@exherbo.org>
 # Distributed under the terms of the GNU General Public License v2
 
-require sourceforge [ project=flac-tools suffix=tar.gz ]
+require sourceforge [ project=flac-tools suffix=tar.gz pnv=${PNV}-release ]
+require autotools [ supported_autoconf=[ 2.5 ] supported_automake=[ 1.16 1.15 1.14 ] ]
```
Comment (tgurr) [inline packages/media-sound/flac123/flac123-0.0.12.exheres-0:5]: for autoconf there's 2.7 and just use only latest if adding it just now, so I'd suggest:
`require autotools [ supported_autoconf=[ 2.7 ] supported_automake=[ 1.16 ] ]`

**Review by tgurr — CHANGES_REQUESTED** [review]
No problem at all, and sorry for e.g. not making use of the GitHub review functions earlier already. Could you please squash your commits to just the version bump? I don't think the details are of much use and rather self-explanatory.

**Comment by Archetypic** [comment]
I have enabled "Allow squash merging" in the fork.

https://postimg.cc/XrzNpqyT

I assume this is insufficient due to your direct request. I will squash, just want confirmation on this.

**Comment by tgurr** [comment]
> I have enabled "Allow squash merging" in the fork.
> 
> https://postimg.cc/XrzNpqyT
> 
> I assume this is insufficient due to your direct request.

Thanks, that's fine, GitHub provides an option to `Squash and merge` on merging the pull request, see:

Screenshot_20210813_092615

Thanks for your contribution and patience!

**Pull request merged**